packages/promise-kit/package.json

```diff
@@ -40,6 +43,7 @@
   "devDependencies": {
     "@endo/eslint-config": "^0.5.1",
     "@endo/ses-ava": "^0.2.33",
+    "@types/node": ">=11.0",
```

Comment:
I'd be ok with bumping this up to `14.0`

Reply:
done. and since that still resolves to 14 that was available in yarn.lock, I also did a yarn upgrade.